Dockerfile.crf

```diff
@@ -74,7 +74,7 @@ ENV TINI_VERSION v0.18.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 ENTRYPOINT ["/tini", "-s", "--"]
-CMD ["./grobid-service/bin/grobid-service", "server", "grobid-service/config/config.yaml"]
+CMD ["./grobid-service/bin/grobid-service"]
```

Comment:
Actually I am wondering, why not 
```
CMD ["./gradlew grobid-service:run"]
```
and forget about the standalone application?

Reply:
Bad idea sorry !